Fix coverage version for py36

bruno-fs opened the pull request.

bruno-fs committed:
Fix coverage version for py36

gtanzillo reviewed (approved):
THe changes seem fine. Not sure about the failure of the 3.9 tests, though

bruno-fs commented:
@gtanzillo 
> THe changes seem fine. Not sure about the failure of the 3.9 tests, though

Mmm I found the problem. Some linting checks seem to be only activated for py3.9

```   
************* Module qpc_tools.utils
  qpc_tools/utils.py:124:24: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
  qpc_tools/utils.py:181:34: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
  qpc_tools/utils.py:182:34: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
  ************* Module qpc_tools.server.install
  qpc_tools/server/install.py:20:0: R0402: Use 'from qpc_tools import server' instead (consider-using-from-import)
  qpc_tools/server/install.py:116:22: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
  ************* Module qpc_tools.cli.install
  qpc_tools/cli/install.py:21:0: R0402: Use 'from qpc_tools import cli' instead (consider-using-from-import)
  qpc_tools/cli/install.py:81:22: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)
```

Because its not so many errors I'll fix them instead of suppressing as we did on quipucords

bruno-fs committed:
Apply linting recomendations

With exception to `consider-using-with`, which would require more code
changes, apply suggestions by pylint.
